### PR TITLE
[profile] log out confirmation screen

### PR DESCRIPTION
**Pull request by arfamomin**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- Added new log out confirmation screen when Log Out is clicked from profile. 
- Removed Impact Fund Header (to be addressed in a future PR)

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

Review the latest commit. See routing change made in Profile/index.tsx. Then look under the Profile/LogOut for new page and styling. Test out logging out and logging back in. Check that styling looks okay.

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

Right now, the width of buttons is flexible but lmk if I should make that a set size too. 

Remove bottom navigation bar from view on this screen (and delete account).

### Online sources
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @ronniebeggs

### CODE REVIEW

File: src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx]: Ideally we should avoid using empty divs to create spacing. If we place the text and the the buttons within the same div, we could use the `rowGap` property.

**Comment by arfamomin** [comment]
If you go to the confirmation page then navigate to one of the other tabs (Cases/Updates) then navigate back to profile, it stays on the confirmation page (as opposed to the profile page). I could find a way to automatically switch back to profile once you leave the confirmation page screen but I think that it won't be an issue once we remove the navigation bar entirely from the confirmation page.

File: src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx]: I don't think we need to wrap the `Text` component with an additional `View`.

File: src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx]: Same here. Opt for vertical padding instead.

**Review by ronniebeggs — CHANGES_REQUESTED** [review]
Looks good so far!

File: src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx]: I noticed that this `buttonView` isn't completely necessary. We should merge the `buttonView` props into the `cancelButton` and `confirmButton`.

File: src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx]: We only need one of these two containers. We should be able to merge their props.

File: src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx]: Make sure to remove commented code before merging.

File: src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx]: Remove this comment here.

File: src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts:6]: In terms of readability, it'd be helpful to organize our style sheets so larger containers are at the top, then component styles follow, and finally text styling at the bottom. This is sort of opinionated but it makes it easier to review if everything is consistent.

**Review by ronniebeggs — CHANGES_REQUESTED** [review]
I left a few more comments. I think we're ready to merge after these are addressed.

File: src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts
Comment (arfamomin) [reply src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts:6]: changed!

**Comment by arfamomin** [comment]
updated based off comments!

**Comment by ronniebeggs** [comment]
> If you go to the confirmation page then navigate to one of the other tabs (Cases/Updates) then navigate back to profile, it stays on the confirmation page (as opposed to the profile page). I could find a way to automatically switch back to profile once you leave the confirmation page screen but I think that it won't be an issue once we remove the navigation bar entirely from the confirmation page.

We can resolve this within the bottom navigation bar logic. I wouldn't worry about it in this PR but thanks for pointing it out!!

File: src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts]: Can accomplish this with `flex: 1`.

File: src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/styles.ts]: Can use `paddingVertical` and `paddingHorizontal` if the dimensions are the same on each side. You can do the same thing for margins as well.

File: src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx]: I consolidated these containers into one. I usually try to avoid nesting divs unless they're explicitly being used for position elements on the screen.

**Review by ronniebeggs — APPROVED** [review]
Looks great! You completed this so fast, I'm sorry it took so long to get around reviewing it. I left a few comments but went ahead and merged them to make it easier. Good job!

**Pull request merged**
